misc/scripts/install_vulkan_sdk_macos.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+# Download and install the Vulkan SDK.
+VULKAN_SDK_VERSION="1.2.189.0"
```

Comment:
It would be better to install the `latest` one so we don't need to bump this version all the time: https://sdk.lunarg.com/sdk/download/latest/mac/vulkan-sdk.dmg